src/compas/utilities/maps.py

```diff
@@ -41,6 +41,10 @@ def geometric_key(xyz, precision=None, sanitize=True):
 
         # 3.142,3.142,3.142
 
+    See also
+    --------
+    geometric_key_xy: Create geometric keys for 2D coordinates
```

Comment:
This will not create a link between the two, but you could, it's something like :meth:`geometry_key_xy` iirc

Reply:
![See Also example](https://user-images.githubusercontent.com/14882117/71204196-d52f1b00-229f-11ea-94c8-b204d9d81839.png)

This works, and the link is correct..

Reply:
OH! cool, didn't know! I thought it needed the whole directive thingy to link properly

Reply:
[Thank Napoleon](https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html#docstring-sections)

:)